scripts/ci/deploy/config/databend-query-node-1.toml

```diff
@@ -48,18 +48,33 @@ dir = "./.databend/logs"
 query_enabled = false
 
 [meta]
-# To enable embedded meta-store, set meta_address to "".
-meta_embedded_dir = "./.databend/meta_embedded_1"
-meta_address = "0.0.0.0:9191"
-meta_username = "root"
-meta_password = "root"
-meta_client_timeout_in_second = 60
+# To enable embedded meta-store, set address to "".
+embedded_dir = "./.databend/meta_embedded_1"
+address = "0.0.0.0:9191"
+username = "root"
+password = "root"
+client_timeout_in_second = 60
 
 # Storage config.
 [storage]
-# fs|s3
+# fs | s3 | azblob
```

Comment:
```suggestion
# fs | s3 | azblob | hdfs
```

Reply:
But this could be complicated. Because we don't have hdfs built in, users may need to compile by themselves with the `hdfs` feature enabled.

Reply:
Considering the actual use of scenarios and release plans, we do not intend to prompt hdfs in the configuration for the time being.